packages/material-ui/src/SwipeableDrawer/SwipeableDrawer.js

```diff
@@ -486,6 +480,12 @@ SwipeableDrawer.propTypes = {
    * @ignore
    */
   variant: PropTypes.oneOf(['permanent', 'persistent', 'temporary']),
+  /**
+   * Defines, from which (average) velocity on, the swipe is
+   * defined as complete although hysteresis isn't reached.
+   * Good threshold is between 0.5 - 2 px/ms
```

Comment:
maybe we could mention, that 0.4 is googles default value?!

Reply:
I haven't realized we could access Android source code. It's smart. I have tried 0.4 on my phone. It's OK. I find 0.1 too low, it's too easy to trigger. 2 starts to be hard to trigger. Maybe we could put the default value at 0.4 and the line-up between 0.2 and 0.8?

Reply:
sounds great!